node/lib/cmd/commit.js

```diff
@@ -118,7 +130,8 @@ exports.executeableSubcommand = co.wrap(function *(args) {
     // this time.
 
     if (repoStatus.areUncommittableSubmodules()) {
-        process.stderr.write(Status.printRepoStatus(repoStatus, relCwd));
```

Comment:
Any particular reason for `process.stderr.write` here, but `console.error` below?